### PR TITLE
fix(tooling): correct schedule property

### PR DESCRIPTION
**Pull request by Maby0**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by francescobrenci-AR — DISMISSED** [review]

File: apps/api/src/server/build-app.js [hunk 1]
```diff
@@ -24,6 +24,17 @@ const buildApp = (
...
+	app.get('/health', async (req, res) => {
+		try {
+			await databaseConnector.$queryRaw`SELECT 1;`;
+		} catch {
+			res.status(500).send('Database connection failed during health check');
+			return;
+		}
+
+		res.status(200).send('OK');
+	});
```
Comment (Maby0) [inline apps/api/src/server/build-app.js:36]: moved health-checks to not require authorisation

Comment (beejjacobs) [reply apps/api/src/server/build-app.js:36]: Good plan. Probably worth a comment in code so people know the order/position is important.

File: azure-pipelines-regenerate-api-key.yml
Comment (beejjacobs) [inline azure-pipelines-regenerate-api-key.yml:1]: Worth removing the `environment` parameter I reckon, as it's not used.

**Review by beejjacobs — COMMENTED** [review]
Looks good. A few minor comments.

Is it worth looking out the KV name that is being processed, too?

**Review by beejjacobs — APPROVED** [review]
Comments resolved, good work!

**Pull request merged**